### PR TITLE
Remove "Machines" tab from MachineConfigPool details page until the API and desired behavior are worked out

### PR DESCRIPTION
**Pull request by TheRealJon**
Fixes https://jira.coreos.com/browse/CONSOLE-1261

### CODE REVIEW

File: frontend/public/components/machine-config-pool.tsx [hunk 1]
```diff
@@ -59,8 +67,8 @@ const MachineConfigPoolCharacteristics: React.SFC<MachineConfigPoolCharacteristi
         <dd>
           {
             configuration.source
-              ? _.map(configuration.source, mc =>
-                <ResourceLink kind={machineConfigReference} name={mc.name} title={mc.name} />
+              ? _.map(configuration.source, (mc, index) =>
+                <ResourceLink key={`configuration-source-${index}`} kind={machineConfigReference} name={mc.name} title={mc.name} />
```
Comment (spadgett) [inline frontend/public/components/machine-config-pool.tsx]: nit: any reason not to use just use `key={index}`?

File: frontend/public/components/machine-config-pool.tsx [hunk 2]
```diff
@@ -37,7 +39,13 @@ const pauseAction = (kind, obj) => ({
 });
 
 const machineConfigPoolReference = referenceForModel(MachineConfigPoolModel);
+const nodeReference = referenceForModel(NodeModel);
 const machineConfigPoolMenuActions = [pauseAction, ...Kebab.factory.common];
+const hasCondition = (mcp: MachineConfigPoolKind, type: MachineConfigPoolConditionType): K8sResourceConditionStatus => {
```
Comment (spadgett) [inline frontend/public/components/machine-config-pool.tsx]: I'd probably call this `conditionStatus`. It sounds like it returns a boolean, but it returns a string

File: frontend/public/components/machine-config-pool.tsx (hunk 1 above)
Comment (TheRealJon) [reply frontend/public/components/machine-config-pool.tsx]: I thought that was bad practice, but I don't really care that much 😆. I can change it.

Comment (spadgett) [reply frontend/public/components/machine-config-pool.tsx]: Yeah, but using the index in a string has the same drawbacks... Unfortunately there's not a good key to use here unless maybe we combine apiVersion+kind+name. I'm not sure it's worth it in this case.

Comment (TheRealJon) [reply frontend/public/components/machine-config-pool.tsx]: Fixed

File: frontend/public/components/machine-config-pool.tsx (hunk 2 above)
Comment (TheRealJon) [reply frontend/public/components/machine-config-pool.tsx]: Fixed

**Pull request merged**